examples/r2i/tensorrt/stb_image.h

```diff
@@ -0,0 +1,8419 @@
+/* stb_image - v2.15 - public domain image loader - http://nothings.org/stb_image.h
```

Comment:
This file is duplicated in tensorflow, tflite and tensorrt. We should refactor it!